ape_foundry/provider.py

```diff
@@ -674,6 +674,18 @@ def set_code(self, address: AddressType, code: ContractCode) -> bool:
         self._make_request("anvil_setCode", [address, code])
         return True
 
+    def set_storage(self, address: AddressType, slot: int, value: HexBytes):
```

Comment:
i noticed it is using a different `HexBytes` here ... core is using `ethpm_types.HexBytes` but this is `hexbytes.HexBytes`.

All tests still pass for me when I make the swap though: https://github.com/ApeWorX/ape-foundry/pull/56